scripts/make_info_card.py

```
@@ -0,0 +1,93 @@
+            group_attrs = ' opacity="0"'
+            animation_elements = (
+                f'    <animate attributeName="opacity" from="0" to="1" begin="{delay}" dur="0.4s" fill="freeze" />\n'
+                f'    <animateTransform attributeName="transform" type="translate" from="0 8" to="0 0" begin="{delay}" dur="0.4s" fill="freeze" />'
+            )
+
+        svg.append(f'  <g class="terminal-text"{group_attrs}>')
+        if animation_elements:
+            svg.append(animation_elements)
+
+        # Draw key and value
+        # Prompt symbol e.g., "❯"
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `❯` (HEAVY RIGHT-POINTING ANGLE QUOTATION MARK ORNAMENT). Did you mean `>` (GREATER-THAN SIGN)?

(RUF003)